packages/next/server/config-shared.ts

```diff
@@ -123,6 +123,7 @@ export interface ExperimentalConfig {
     layoutRaw: boolean
     remotePatterns: RemotePattern[]
     unoptimized?: boolean
+    future?: boolean
```

Comment:
I wonder if this being a config may make users think we automatically map `next/image` -> `next/future/image`

Reply:
Maybe `allowFutureImage`?

Reply:
Done in [7b0c3d2](https://github.com/vercel/next.js/pull/37927/commits/7b0c3d2cde4ffcac409a1b87312dd4eb018b6fbe)